lib/validators/specResolver.ts

```diff
@@ -990,9 +932,9 @@ export class SpecResolver {
   private buildOneOfReferences(rootNode: PolymorphicTree): Set<PolymorphicTree> {
     let result = new Set()
     result.add({ $ref: `#/definitions/${rootNode.name}` })
-    for (const entry of rootNode.children.entries()) {
-      if (entry[1]) {
-        result = new Set([...result, ...this.buildOneOfReferences(entry[1])])
+    for (const en of rootNode.children.entries()) {
```

Comment:
>en [](start = 15, length = 2)

can we use `for ( [enName, enObj]...)` for more clarity

Reply:
we even can use `values()` instead of `entries()`.